src/process.ts

```diff
@@ -0,0 +1,63 @@
+function zip(firstArray: string[], secondArray: string[]) {
+  let str = "";
+  for (let i = 0; i < firstArray.length; i++) {
+    if (secondArray[i]) {
+      str += firstArray[i] + secondArray[i];
+    } else {
+      str += firstArray[i];
+    }
+  }
+
+
+
+  return str;
+}
+
+export interface RowObject {
+  [key: string]: string | number;
+}
+
+export default function createProcess(customProcessor: Function, resolver: Function) {
```

Comment:
@awmuncy What do you think about calling it `createProcessor`?
Might be good to avoid using the word "process"?

Reply:
Good point, I do think that is a better name. I've updated it.